test/plugin.spec.js

```diff
@@ -313,9 +312,11 @@ test.serial('should keep [hash] in importScripts after configuring SW', async t
   const plugin = new SWPrecacheWebpackPlugin({filepath, importScripts: ['some_sw-[hash].js']});
 
   plugin.apply(compiler);
-  compiler.plugin('after-emit', (compilation) => {
+  compiler.plugin('after-emit', (compilation, callback) => {
     plugin.configure(compiler, compilation);
+    callback();
```

Comment:
From what I'm understanding - each compiler must inform that it has ended by calling callback.

Without this ```await runCompiler``` hungs because this plugin never called callback to indicate it has finished . This previously worked because SWPrecacheWebpackPlugin has been calling callback two times and webpack has seen that there are no callbacks that have not been called and after-emit phase has finished.